source/cppfront.cpp

```diff
@@ -72,7 +72,7 @@ auto main(
             }
             else {
                 std::cerr << arg.text << " - ambiguous compiler flag name, did you mean one of these?\n";
-                for (auto a : ambiguous) {
+                for (auto const &a : ambiguous) {
```

Comment:
```suggestion
                for (auto &&a : ambiguous) {
```

Reply:
I'm not sure moving from std::string is in any way better, due to small string optimizations, but I'm not opposed to this change either.

Reply:
Again, I don't see it as optimization, more as making it more flexible. I think in the current form the binary result will be identical.

Reply:
applied the & next to const as in the rest of code